lib/jxl/dec_cache.h

```diff
@@ -205,6 +209,12 @@ struct PassesDecoderState {
                                       kGroupDim + 2 * kGroupDataYBorder);
       }
     }
+    if (fast_xyb_srgb8_conversion) {
+      for (size_t _ = fixpoint_srgb_buffer.size(); _ < num_threads; _++) {
```

Comment:
Would it make sense to `reserve`?

Reply:
I'd say probably not -- but I also think I should actually wait to refactor things a bit before I submit this....